### PR TITLE
AOAF-422

### PR DESCRIPTION
**Pull request by ch-skathirvel**
Desc : Update redirect url with home url once the presenter account submitted.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/middleware/authentication.middleware.ts [hunk 1]
```diff
@@ -2,13 +2,23 @@ import { NextFunction, Request, Response } from "express";
 import { authMiddleware, AuthOptions } from "@companieshouse/web-security-node";
 import { env } from "../config";
 import { removeLangFromUrl } from "../utils/query";
-
+import { Urls, PrefixedUrls } from "../constants";
 export const authenticationMiddleware = (req: Request, res: Response, next: NextFunction) => {
     const CHS_URL = env.CHS_URL;
+    const RETURN_URL = getReturnCallbackURL(req);
```
Comment (amartin7663) [inline src/middleware/authentication.middleware.ts:8]: This local variable should be in camel case

Comment (ch-skathirvel) [reply src/middleware/authentication.middleware.ts]: Fixed. Can you please review again?

**Review by amartin7663 — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**